src/components/info-tile/info-tile.scss

```diff
@@ -0,0 +1,344 @@
+/**
+* @prop --info-tile-border-radius: defines the radius of corners of the info-tile. Defaults to `1rem`
+* @prop --info-tile-icon-color: defines the fill color of the info-tile icon. Defaults to `--contrast-1000`
+* @prop --info-tile-text-color: defines the color of the info-tile label. Defaults to `--contrast-1100`
+* @prop --info-tile-background-color: defines the backgrounds color of the info-tile icon. Defaults to `--contrast-100`
+* @prop --info-tile-badge-text-color: Text color of the notification badge. Defaults to `--color-white`
+* @prop --info-tile-badge-background-color: Background color of the notification badge. Defaults to `--color-red-default`
+* @prop --info-tile-progress-fill-color: Determines the color of the progressed section. Defaults to `--lime-primary-color`.
+* @prop --info-tile-progress-background-color: Determines the background color of the central section of the progress bar. Defaults to `--info-tile-background-color`.
+* @prop --info-tile-progress-suffix-color: Determines the color of the progress prefix. Defaults to `--contrast-1000`.
+ * @prop --info-tile-progress-text-color: Determines the color of the progress value. Defaults to `--info-tile-text-color`.
+ * @prop --info-tile-progress-prefix-color: Determines the color of the progress suffix. Defaults to `--contrast-1000`.
+*/
+
+@use '../../style/mixins';
+
+:host(limel-info-tile) {
+    --badge-text-color: var(
+        --info-tile-badge-text-color,
+        rgb(var(--color-white))
+    );
+    --badge-background-color: var(
+        --info-tile-badge-background-color,
+        rgb(var(--color-red-default))
+    );
+
+    --circular-progress-text-color: var(
+        --info-tile-progress-text-color,
+        var(--info-tile-text-color)
+    );
+    --circular-progress-suffix-color: var(--info-tile-progress-suffix-color);
+    --circular-progress-prefix-color: var(--info-tile-progress-prefix-color);
+    --circular-progress-track-color: rgb(var(--contrast-800), 0.3);
+    --circular-progress-fill-color: var(--info-tile-progress-fill-color);
+    --circular-progress-background-color: var(
+        --info-tile-progress-background-color,
+        var(--info-tile-background-color)
+    );
+
+    --label-min-size: 0.75rem;
+    --label-preferred-size: 6cqw;
+    --label-max-size: 1rem;
+
+    --value-min-size: 1rem;
+    --value-preferred-size: 20cqw;
+    --value-max-size: 4rem;
+
+    --suffix-prefix-min-size: 0.75rem;
+    --suffix-prefix-preferred-size: 8cqw;
+    --suffix-prefix-max-size: 1.5rem;
+
+    --icon-min-size: 2rem;
+    --icon-preferred-size: 60cqh;
+    --icon-max-size: calc(100cqw - 0.5rem);
+
+    isolation: isolate;
+    container-type: size;
+    position: relative;
+    display: flex;
+
+    width: 100%;
+    height: 100%;
+
+    * {
+        box-sizing: border-box;
+    }
+}
+
+:host(limel-info-tile[disabled]) {
+    a {
+        opacity: 0.5;
+        cursor: not-allowed;
```

Comment:
This doesn't seem great? Didn't you, @Kiarokh, say that an info tile can be used for only illustrating something, and thus not have a url? In that case, we wouldn't want to dim the tile and show a `not-allowed`-cursor when hovering it? 🤔

Reply:
Not sure if I understand. This is specifically for when it is `disabled`. So styles shouldn't be triggered, for "visualization mode only".

A disabled `<a` on the other hand doesn't really exist, theoretically; unless you are dealing with CRM and permissions user rights and so forth. So I couldn't decide what to do.

Reply:
Ah! Sorry, I was confused. I noticed at some point that the link was automatically set to `'#'` when `disabled` is true, and when I came across this a good deal later, I was thinking that `disabled` was set if the link is left empty. Just a complete mixup in my head. Move along, nothing to see here! 😝